fastapi_sqla/_pytest_plugin.py

```diff
@@ -92,19 +96,19 @@ def sqla_modules():
 
 
 @fixture
-def sqla_reflection(sqla_modules, sqla_connection, db_url):
+def sqla_reflection(sqla_modules, sqla_connection):
     import fastapi_sqla
 
     fastapi_sqla.Base.metadata.bind = sqla_connection
     fastapi_sqla.Base.prepare(sqla_connection.engine)
 
 
 @fixture
-def patch_engine_from_config(request, db_url, sqla_connection, sqla_transaction):
+def patch_engine_from_config(request, sqla_connection, sqla_transaction):
     """So that all DB operations are never written to db for real."""
     from fastapi_sqla.sqla import _Session
 
-    if "dont_patch_engines" in request.keywords:
+    if "dont_patch_engines" in request.keywords:  # pragma: no cover
```

Comment:
We weren't actually testing it, and it was flagged as not covered after I cleaned up `test_startup`. I don't think it should be that PR responsibility to add coverage for this case in this fixture

Reply:
🤔 weird that it was flagged as not covered as that marker is used in tests.

Reply:
(Anyway, agree it is not the responsibility of current PR to cover this)